coordinator/src/node.rs

```diff
@@ -134,7 +135,14 @@ impl Node {
     }
 
     pub async fn trade(&self, trade_params: &TradeParams) -> Result<()> {
-        match self.decide_trade_action(&trade_params.pubkey)? {
+        match spawn_blocking({
```

Comment:
🔧 I think `trade` should be sync. I'm proposing the change in a separate PR.

Reply:
I'll have a look then.